### PR TITLE
Add test support for floor_30_minutes_dt method also added test_utils…

### PR DESCRIPTION
**Pull request by BodaleDenis**
… file which contain utils methods for testing

# Pull Request

## Description

- I've added test support for floor_30_minutes_dt method against all possible minutes [0,59] range
- I've added new file test_utils.py which can serve as a place where are written code in order to prepare the setup for tests
- I've added Docstrings for newly added code in order to explain better

Fixes issue #5 

## How Has This Been Tested?

_Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration

I've added tests, not functional code

- [ ] No
- [ x] Yes

## Checklist:

- [x ] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [ x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ x] I have added tests that prove my fix is effective or that my feature works
- [x ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/tests/test_main.py [hunk 1]
```diff
@@ -17,3 +17,18 @@ def test_read_latest():
...
+    For minutes in range [30, 60) => Will floor to 30 minutes
+    """
+    list_of_time = get_every_minute()
+    for time in list_of_time:
+        floor_minute = floor_30_minutes_dt(time)
+        if time.minute < 30:
+            assert floor_minute.minute == 0
+        else:
+            assert floor_minute.minute == 30
+
```
Comment (peterdudfield) [inline src/tests/test_main.py:34]: I really like this, thanks so much for doing this. This should cover all times which is amazing. 

Would you be able to add one other example that shows it explicity i.e 

`
in = datetime.fromisoformat('2019-01-01 12:15:05')
out = datetime.fromisoformat('2019-01-01 12:00:00')
assert floor_30_minutes_dt(in) == out 
`

- or something like that

File: src/tests/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+from datetime import datetime, timezone, timedelta
```
Comment (peterdudfield) [inline src/tests/test_utils.py:1]: looks great!

File: src/tests/test_main.py (hunk 1 above)
Comment (BodaleDenis) [reply src/tests/test_main.py:34]: Yes I can do explictly too if you want, but I will make another pull request when I have some time.
I'm also Software engineer full time so I do contribute in my spear time 😄

Comment (peterdudfield) [reply src/tests/test_main.py:34]: No problem - I totally understand - Thank you for doing this. I'll add comment to the issue.

**Pull request merged**